packages/edit-site/src/components/global-styles/README.md

```diff
@@ -15,77 +15,3 @@ function MyComponent() {
 	return <GlobalStylesUI />;
```

Comment:
Any thoughts on the UI? can we share with block inspector UI at some point?

Reply:
It should be possible I think to create high level UI components that live in `@wordpress/block-editor` and are used by both the block inspector and global styles. Ideally only the `value` and `onChange` differs. So basically what you propose in https://github.com/WordPress/gutenberg/discussions/37064. I note that since you opened that discussion we've come a long way in making the UIs look very similar so ideally now we can make the shared components be very high level (e.g. `TypographyPanel`, `DimensionsPanel`).

I'll make a note to explore this!